SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -38,8 +39,8 @@ struct ViewListingView: View {
                 
                 GeometryReader { geometry in
                     ImageCarouselView(numberOfImages: self.numberOfImages) {
-                        ForEach([self.image, UIImage(named: "ProfilePhotoPlaceholder")!, UIImage(named: "ProfilePhotoPlaceholder")!], id:\.self) { image in
-                            Image(uiImage: image)
+                        ForEach(images, id:\.self) { image in
+                            Image(uiImage: image ?? UIImage())
```

Comment:
Maybe if the image doesn't exist for some reason, use UIImage(named:"placeholder") instead of just a blank image